app/src/full/java/io/homeassistant/companion/android/sensors/GeocodeSensorManager.kt

```diff
@@ -98,14 +102,19 @@ class GeocodeSensorManager : SensorManager {
                     "administrative_area" to it.adminArea,
                     "country" to it.countryName,
                     "iso_country_code" to it.countryCode,
+                    "language" to it.locale.displayLanguage,
```

Comment:
This always seems to match the device language, is that useful information to have here?

Reply:
If you leave the country it should update to match the visiting locale. At least that's how I took the description lol 

https://developer.android.com/reference/java/util/Locale#getDisplayLanguage()

https://developer.android.com/reference/android/location/Address#getLocale()

Reply:
I think it matches the locale used to generate the Address object. Otherwise changing the device language shouldn't change this value, but it does.

For example, I'm in The Netherlands and have my device set to English. The geocoder shows `language` right now as "English". If your idea was correct that should be "Dutch", correct? If I change the device language to German it'll show "Deutsch" for `language`, etc.

Reply:
Ah that is strange, I must've misunderstood it when it said `Returns the Locale associated with this address.` I took it as it applies to that `address` like all other attributes lol. Will remove this attribute then :)